Cargo.toml

```diff
@@ -44,6 +44,10 @@ harness = false
 name = "dns"
 harness = false
 
+[[example]]
```

Comment:
No need for this example. Instead you can add a test using the `#[cfg(test)]` option. Look inside `dns.rs` for how to add that.